src/client/theme-default/index.ts

```diff
@@ -1,4 +1,8 @@
 import './styles/fonts.css'
 
+import VPHomeContent from './components/VPHomeContent.vue'
+
 export * from './without-fonts'
 export { default as default } from './without-fonts'
+
+export { VPHomeContent }
```

Comment:
export it directly in from without-fonts file, see how others are exported there